File: app/src/main/java/ms/imagine/foodiemate/presenters/FbDatabasePresenter.java
package ms.imagine.foodiemate.presenters;

import com.google.firebase.database.DatabaseReference;
import com.google.firebase.database.FirebaseDatabase;

public class FbDatabasePresenter {
    protected FirebaseDatabase fbInstance;
    protected DatabaseReference firebaseDB;

    FbDatabasePresenter(String _uid) {
        fbInstance = FirebaseDatabase.getInstance();
        firebaseDB = fbInstance.getReference().child(_uid);
    }
}

File: app/src/main/java/ms/imagine/foodiemate/presenters/FbDatabasePresenter.kt
package ms.imagine.foodiemate.presenters

import com.google.firebase.database.DatabaseReference
import com.google.firebase.database.FirebaseDatabase

open class FbDatabasePresenter(_uid: String?){
    var fbInstance:FirebaseDatabase = FirebaseDatabase.getInstance()
    var firebaseDB:DatabaseReference = fbInstance.reference.child(_uid)

}